apps/pie-docs/src/assets/styles/components/_contentPage.scss

```diff
@@ -170,7 +170,11 @@
         margin-bottom: f.spacing(i);
     }
 }
-.c-contentPage-content a {
+
+.c-content-container > p a,
+.c-content-container > span a,
+.c-content-container > ul a,
+.c-notification-message a {
     color: f.$color-content-link-distinct;
```

Comment:
I understand we need to do that to avoid specificity issues but I'm afraid this will get out of sync easily.

have you considered moving/nesting `.c-card-label` styles in `.c-card-labelContainer`? I think this will have a higher specificity and will apply the correct color for anchor tags without changing other files.

Reply:
thank you!